loadtest/control/simulcontroller/controller.go

```diff
@@ -162,6 +162,10 @@ func (c *SimulController) Run() {
 			run:       searchPosts,
 			frequency: 0.1,
 		},
+		{
+			run:       c.createPostReminder,
+			frequency: 0.1,
```

Comment:
This is just made-up for now. We'll replace this with the right number once the feature is live on community.

Reply:
Should we create a ticket so we don't forget?

Reply:
Done: https://mattermost.atlassian.net/browse/MM-46110